packages/bruno-cli/src/commands/run.js

```diff
@@ -165,6 +167,73 @@ const getCollectionRoot = (dir) => {
   return collectionBruToJson(content);
 };
 
+const makeJunitOutput = async (results, outputPath) => {
+  const output = {
+    testsuites: {
+      testsuite: []
+    }
+  };
+
+  results.forEach((result, idx) => {
+    const assertionTestCount = result.assertionResults ? result.assertionResults.length : 0;
+    const testCount = result.testResults ? result.testResults.length : 0;
+    const totalTests = assertionTestCount + testCount;
+
+    const suite = {
+      '@name': result.suitename,
+      '@errors': 0,
+      '@failures': 0,
+      '@skipped': 0,
+      '@tests': totalTests,
+      '@timestamp': new Date().toISOString().split('Z')[0],
+      '@hostname': os.hostname(),
+      '@time': result.runtime.toFixed(3),
+      testcase: []
+    };
+
+    result.assertionResults &&
+      result.assertionResults.forEach((assertion) => {
+        const testcase = {
+          '@name': `${assertion.lhsExpr} ${assertion.rhsExpr}`,
```

Comment:
How about adding the operator to the name, I think that is important to see the actual assertion that is tested here.

Reply:
In testing this it seems like the operator is already included in the `rhsExpr`? When I ran this against some of my existing test suites I got results like `res.body.array_field.length gt 0` or `res.body[0].field_name isString`. Are there some conditions where the operator isn't included in the rhsExpr?